spec/features/user_creates_a_draft_moment_spec.rb

```diff
@@ -77,6 +77,10 @@
         find('#submit').click
       end
 
+      within('#moment_strategy_accordion') do
+        find('.accordion').click
```

Comment:
Could you explain why you added this? We open the accordion but we aren't testing for anything.  I think this can be removed.

Reply:
Hey @julianguyen  so I had a look!

This code snippet closes the accordion that was opened on line 68. 

The test would still pass without this code as well, the accordion could be left open it makes no difference to the behaviour, but if you look at line 47 and line 63 the previously opened accordions are closed in the exact same way. 

So do you want me to remove it or should I just leave it in?

Reply:
I think it's safe to remove since we're not testing anything related to the accordion being closed!

Reply:
Done :)